runtime/autoload/man.vim

```
@@ -135,7 +135,15 @@ function! s:get_page(path) abort
   let manwidth = empty($MANWIDTH) ? winwidth(0) : $MANWIDTH
   " Force MANPAGER=cat to ensure Vim is not recursively invoked (by man-db).
   " http://comments.gmane.org/gmane.editors.vim.devel/29085
-  return s:system(['env', 'MANPAGER=cat', 'MANWIDTH='.manwidth, 'man', a:path])
+  try
+    if !has('win32') && system('uname -s') =~ 'Darwin'
+      return s:system(['env', 'MANPAGER=cat', 'MANWIDTH='.manwidth, 'man', a:path])
+    else
+      return s:system(['env', 'MANPAGER=cat', 'MANWIDTH='.manwidth, 'man', '-l', a:path])
+    endif
+  catch /E145:/
+    " Restricted mode
```

What's the effect of this? Showing an error is better than silently failing.